Alpaca.Markets/AlpacaTradingClient.General.cs

```diff
@@ -154,6 +154,39 @@ public sealed partial class AlpacaTradingClient
                 _alpacaRestApiThrottler, builder, cancellationToken);
         }
 
+        /// <summary>
+        /// Gets portfolio equity history from Alpaca REST API endpoint.
+        /// </summary>
+        /// <param name="startDate">Start date for desired history.</param>
+        /// <param name="endDate">End date for desired history.</param>
+        /// <param name="period">Period value for desired history.</param>
+        /// <param name="timeframe">Timeframe value for desired history.</param>
+        /// <param name="extendedHours">Whether or not the history view should include extended hours.</param>
+        /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+        /// <returns>Read-only portfolio history information object.</returns>
+        public Task<IPortfolioHistory> GetPortfolioHistoryAsync(
+                DateTime? startDate,
+                DateTime? endDate,
+                HistoryPeriod? period,
+                HistoryTimeframe? timeframe,
+                Boolean? extendedHours,
```

Comment:
I checked it out and it looks like the default is today with "1m" timeframe. The default behavior is a bit tricky to explain because I believe that you can specify any combination of two of the three (start & end date / timeframe / period.) But I'll try to make the comment clearer.